### ISSUE TITLE
Can i know why of this and how to fix it? I can´t debug.

### ISSUE THREAD

**Issue by Gameadictive** [microsoft/WinDbg-Feedback#211]
*** WARNING: Unable to verify checksum for v8-9.3.345.16.dll
*** WARNING: Unable to verify checksum for mono-2.0-sgen.dll
*** WARNING: Unable to verify checksum for libcef.dll
*** WARNING: Unable to verify timestamp for xaudio2_7.dll

**Comment by lzybkr** [comment]
Those warnings are informational and shouldn't block you from debugging. Can you elaborate on what isn't working?

**Comment by Gameadictive** [comment]
Yes, when i debug something it doesn´t tell me what or where is exactly the problem, always is a error in memory access.

**Comment by lzybkr** [comment]
You'll have to provide a lot more detail before we can offer any help.

**Comment by KenSykes** [comment]
We are cleaning up the issues list. This bug is older than six months. If you still see this issue in the current debugger and would like to see a fix please submit a new report.\ Thanks!
